sphinxval/utils/validation.py

```diff
@@ -446,21 +449,27 @@ def fill_dict_row(sphinx, dict, energy_key, thresh_key, profname_dict):
     #MATCHING INFORMATION - cast all matching info to strings to avoid problems
     #with read/write. Kept mainly for human reference and traceability. Not used
     #in the validation process.
+    dict["All Thresholds in Prediction"].append(str(sphinx.prediction.all_thresholds))
     dict["Last Eruption Time"].append(str(sphinx.last_eruption_time))
     dict["Last Trigger Time"].append(str(sphinx.last_trigger_time))
     dict["Last Input Time"].append(str(sphinx.last_input_time))
     
     try:
         dict["Threshold Crossed in Prediction Window"].append(str(sphinx.threshold_crossed_in_pred_win[thresh_key]))
+        tc = [str(x) for x in sphinx.all_threshold_crossing_times[thresh_key]]
+        dict["All Threshold Crossing Times"].append(str(tc))
```

Comment:
Note that often times `str(obj)` and `repr(obj)` produce the same result, but the intention is for `repr(obj)` to be more comprehensive and `str(obj)` to be more human-readable (e.g. concatenated array)

Reply:
Are you requiring the change to repr? I will note that this can be used. I didn't know about that function.
Also, if there is an .append it is because I definitely need an .append.